src/components/Header/Header.tsx

```diff
@@ -0,0 +1,121 @@
+import {
+  FC,
+  Dispatch,
+  SetStateAction,
+  useState,
+  useRef,
+  useEffect,
+} from 'react';
+import cn from 'classnames';
+
+import { Todo } from '../../types/Todo';
+import { ErrorMessages } from '../../types/ErrorMessages';
+import { postTodo, USER_ID } from '../../api/todos';
+
+interface Props {
+  setTempTodo: Dispatch<SetStateAction<Todo | null>>;
+  setTodos: Dispatch<SetStateAction<Todo[]>>;
+  onSetError: (newError: ErrorMessages) => void;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  onTodosChange: any;
```

Comment:
1) don't disable eslint!
2) don't use any
```suggestion
  onTodosChange: (newTodos: Todo[]) => void;
```